### ISSUE TITLE
Typescript types do not allow using regex

### ISSUE THREAD

**Issue by F7502** [kapouer/express-urlrewrite#9]
When I am using `rewrite` with a regex then typescript complains that the first parameter must be a string:
```ts
app.use(rewrite(/^\/i(\w+)/, '/items/$1'));
```
this gives this error when running `tsc`:
```
error TS2345: Argument of type 'RegExp' is not assignable to parameter of type 'string'.
```

**Comment by kapouer** [comment]
Can you check with latest commit ?

**Comment by F7502** [comment]
Works fine now. That was fast, many thanks :-)

**Comment by kapouer** [comment]
Fixed in 2.0.1
